spec/spec.md

```diff
@@ -902,21 +902,90 @@ Get all objects of a given schema type:
 
 `RecordsDelete` messages are JSON objects that include general [Message Descriptor](#message-descriptors) properties and the following additional properties, which ****must**** be composed as follows:
 
-- The message object ****MUST**** contain a `recordId` property, and its value ****MUST**** be the `recordId` of the logical record the entry corresponds with.
 - If the message object is attached to a Protocol, and its value ****MUST**** be a [_Computed Context ID_](#computed-context-ids). If the message is not attached to a Protocol, it ****MUST NOT**** contain a `contextId` property.
```

Comment:
I don't think this is meant for Deletes, since the delete is targeting a specific `recordId` it doesn't need any further information/context.

```suggestion
```